### PR TITLE
[GCI-200]Add a private constructor to FilterUtil

### PR DESCRIPTION
**Pull request by Amritanshu54**
Added private constructor (line 36) - GCI 2017-18 task (https://issues.openmrs.org/browse/GCI-200)

I added a private constructor to line 36.

## Issue I worked on

The issue - https://issues.openmrs.org/browse/GCI-200
JIRA ticket - https://issues.openmrs.org/browse/GCI-235

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by merovingienne** [comment]
@Amritanshu54 the code looks okay :) The constructor could be a one-liner, but that's just nitpicking. Also, could you kindly follow the instructions below? Specifically the PR title & the commit message.
https://wiki.openmrs.org/display/docs/Pull+Request+Tips

Please squash the commits into a single commit with the proper message (as per above) if you push to the same PR.

**Comment by Amritanshu54** [comment]
Added the required snippet into the code. 
For the GCI task, visit - https://codein.withgoogle.com/dashboard/task-instances/5637348442243072/

File: web/src/main/java/org/openmrs/web/filter/util/FilterUtil.java [hunk 1]
```diff
@@ -28,6 +28,9 @@
  */
 public class FilterUtil {
 	
+	private FilterUtil() {
+}
```
Comment (gayanW) [inline web/src/main/java/org/openmrs/web/filter/util/FilterUtil.java]: `}` is not aligned properly.

**Comment by gayanW** [comment]
Branch name should be equal to ticket name. But in your case it is `Amritanshu54:Amritanshu54-patch-1`

I think you'll also need update the description above. The Jira ticket, and Issue points to two different links.

**Pull request closed without merging**